.github/workflows/verify.yaml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
   push:
   pull_request:
-    branches: [ main ]
+    branches: [ main, refactor/* ]
```

Comment:
I guess this won't be needed anymore, or do we want to keep it for further needs?

Reply:
i thought about it, it could be useful going forward. in any case, we'll need it until this PR is merged, because the PR head is not on my fork but in the EDC repo.

Reply:
i don't think, because the branches listed here are the destination ones, and for this PR it is `main`, anyway I'm ok with keeping it.

Reply:
uh, i think you're right!